packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -125,6 +125,8 @@ export const MEDIA_UPDATE_TYPE = {
   * @property {String} audio.deviceId
   * @property {Object} video
   * @property {String} video.deviceId
+  * @property {String} video.localVideoQuality // [240p, 360p, 480p, 720p, 1080p] // we can add _1, _2 for more options like
+  * https://docs.agora.io/en/Video/API%20Reference/web_ng/globals.html#videoencoderconfigurationpreset
```

Comment:
come on guys, we cannot have links to agora docs in our API

Reply:
Sorry @marcin-bazyl, We missed this in our internal review, We will remove the link and commit the changes.
@arun3528, @meta-dipanshu-sharma, @rajeshtezu90

Reply:
I have removed the agora links.